README.md

```diff
@@ -19,9 +33,7 @@ Play SOAP is an sbt plugin that transforms WSDLs into SOAP client interfaces, an
 To install sbt WSDL into your Play project, add the following lines to your `project/plugins.sbt`:
 
 ```scala
-resolvers += Resolver.url("play-sbt-plugins", url("https://dl.bintray.com/playframework/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
```

Comment:
This resolver is not needed anymore (bintray is offline). Correct is https://repo.scala-sbt.org/ now, which is build in sbt anyway, see https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.typesafe.sbt/sbt-play-soap/scala_2.10/sbt_0.13/